src/Resources/config/app/config.yml

```diff
@@ -62,7 +62,7 @@ sylius_grid:
             driver:
                 name: doctrine/orm
                 options:
-                    class: Sylius\RefundPlugin\Entity\CreditMemo
+                    class: %sylius_refund.model.credit_memo.class%
```

Comment:
AFAIK this parameter is never set inside the plugin.

Reply:
I guess it's the SyliusResourceBundle, which sets this parameter, but I'm not sure about that. But the parameter is definitely available in the container.

Reply:
I guess no? https://github.com/Sylius/SyliusResourceBundle/search?q=sylius_refund.model.credit_memo.class&unscoped_q=sylius_refund.model.credit_memo.class

Reply:
Magic happens here https://github.com/Sylius/SyliusResourceBundle/blob/3464579ed05b199fcd8bc12e5444b36d65d6a4a5/src/Bundle/DependencyInjection/Driver/AbstractDriver.php#L48 🙂

Reply:
![](https://media3.giphy.com/media/wMsD35WIjjUFq/giphy.webp?cid=ecf05e47c93f146a0da9c9315b7a51673f3b313f27e1512c&rid=giphy.webp)